GitTfs.Vs2015/TfsHelper.Vs2015.cs

```diff
@@ -13,7 +13,7 @@ public TfsHelper(TextWriter stdout, TfsApiBridge bridge, IContainer container)
 
         protected override string GetDialogAssemblyPath()
         {
-            var tfsExtensionsFolder = TryGetUserRegString(@"Software\Microsoft\VisualStudio\14.0\ExtensionManager\EnabledExtensions", "Microsoft.VisualStudio.TeamFoundation.TeamExplorer.Extensions,14.0.23102");
+            var tfsExtensionsFolder = TryGetUserRegStringStartingWithName(@"Software\Microsoft\VisualStudio\14.0\ExtensionManager\EnabledExtensions", "Microsoft.VisualStudio.TeamFoundation.TeamExplorer.Extensions");
```

Comment:
When I looked at this, I found having to think hard about how `""Microsoft.VisualStudio.TeamFoundation.TeamExplorer.Extensions"` was going to match things, even though it should be pretty obvious. How do you feel about a comment right before this line with examples of the keys that should match? Or do you think it would make sense to change `TryGetUserRegStringStartingWithName(string path, string name)` to `TryGetUserRegString(string path, Regex pattern)`, so the wildcard would be explicit in the argument?